### ISSUE TITLE
Train scrfd very low

### ISSUE THREAD

**Issue by ngocson1804** [deepinsight/insightface#1749]
I retrain scrfd model with scrfd_10g_bnkps.py config file on 4 A100 gpus. But training speed is very low. The log file show "lr: 1.666e-05, eta: 76 days, 3:05:39, time: 0.565, data_time: 0.139, memory: 8009, loss_cls: 0.0006, loss_bbox: 2.0173, loss_kps: 2.2101, loss: 4.2280". How long does it take you to train the model?

**Comment by nttstar** [comment]
2021-03-18 10:23:05,709 - mmdet - INFO - Epoch [2][100/403]     lr: 3.353e-03, eta: 1 day, 14:58:23, time: 0.818, data_time: 0.459, memory: 2649, loss_cls: 0.3303, loss_bbox: 0.8845, loss_kps: 0.
3281, loss: 1.5429

**Comment by ngocson1804** [comment]
I see the training process already running on GPUs. But it still takes more than 70days. Do you have any suggesstion?

**Comment by ledaiduongvnth** [comment]
@ngocson1804 There are several reasons :
- Your infrastructure, you had better to store your training data on a SSD mounted disk directly to the DGX rather than a network storage.
- Your base docker image(in the case that you are using docker to train). it is recommended to use this image https://ngc.nvidia.com/catalog/containers/nvidia:pytorch and the newest version of pytorch
- A100 sm_80 architecture is suitable with this version of torch 
`pip3 install torch==1.9.0+cu111 torchvision==0.10.0+cu111 torchaudio==0.9.0 -f https://download.pytorch.org/whl/torch_stable.html`
